srcpkgs/kdenlive/template

```diff
@@ -9,16 +9,16 @@ hostmakedepends="
 makedepends="
  kdeclarative-devel kfilemetadata5-devel knewstuff-devel knotifyconfig-devel
  kplotting-devel mlt-devel qt5-multimedia-devel qt5-webkit-devel purpose-devel
- v4l-utils-devel ksolid-devel qt5-quickcontrols2-devel"
-depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit qt5-quickcontrols
- qt5-quickcontrols2 vlc"
+ v4l-utils-devel ksolid-devel qt5-quickcontrols2-devel qt5-networkauth-devel"
+depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit qt5-quickcontrols2
+ qt5-networkauth vlc"
```

Comment:
```suggestion
depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit vlc"
```

The idea with SONAMES is that the dependency on those is detected automatically, so they don't need to be listed here.

Reply:
oh i see.. i understand.

Reply:
even though it's been removed (`qt5-quickcontrols2` and `qt5-networkauth`), there's still `SONAME:`.
but looks normal. and kdenlive runs properly. 
```
=> kdenlive-21.04.3_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libKF5KIOWidgets.so.5 <-> kio>=5.26.0_1
   SONAME: libmlt++.so.3 <-> mlt>=6.4.1_1
   SONAME: libQt5Gui.so.5 <-> qt5-gui>=5.15.2_3
   SONAME: libQt5Core.so.5 <-> qt5-core>=5.15.2_3
   SONAME: libstdc++.so.6 <-> libstdc++>=4.4.0_1
   SONAME: libc.so.6 <-> glibc>=2.32_1
   SONAME: libQt5Widgets.so.5 <-> qt5-widgets>=5.15.2_3
   SONAME: libQt5DBus.so.5 <-> qt5-dbus>=5.15.2_3
   SONAME: libQt5Xml.so.5 <-> qt5-xml>=5.15.2_3
   SONAME: libKF5KIOFileWidgets.so.5 <-> kio>=5.26.0_1
   SONAME: libKF5Bookmarks.so.5 <-> kbookmarks>=5.26.0_1
   SONAME: libKF5NotifyConfig.so.5 <-> knotifyconfig>=5.26.0_1
   SONAME: libKF5NewStuff.so.5 <-> knewstuff>=5.26.0_1
   SONAME: libKF5XmlGui.so.5 <-> kxmlgui>=5.26.0_1
   SONAME: libKF5Notifications.so.5 <-> knotifications>=5.26.0_1
   SONAME: libKF5TextWidgets.so.5 <-> ktextwidgets>=5.26.0_1
   SONAME: libKF5Declarative.so.5 <-> kdeclarative>=5.26.0_1
   SONAME: libKF5KIOGui.so.5 <-> kio>=5.36.0_1
   SONAME: libKF5KIOCore.so.5 <-> kio>=5.26.0_1
   SONAME: libKF5Crash.so.5 <-> kcrash>=5.26.0_1
   SONAME: libKF5Service.so.5 <-> kservice>=5.26.0_1
   SONAME: libKF5DBusAddons.so.5 <-> kdbusaddons>=5.26.0_1
   SONAME: libKF5JobWidgets.so.5 <-> kjobwidgets>=5.26.0_1
   SONAME: libKF5Completion.so.5 <-> kcompletion>=5.26.0_1
   SONAME: libKF5IconThemes.so.5 <-> kiconthemes>=5.26.0_1
   SONAME: libKF5Archive.so.5 <-> karchive>=5.26.0_1
   SONAME: libKF5ItemViews.so.5 <-> kitemviews>=5.26.0_1
   SONAME: libKF5ConfigWidgets.so.5 <-> kconfigwidgets>=5.26.0_1
   SONAME: libKF5WidgetsAddons.so.5 <-> kwidgetsaddons>=5.26.0_1
   SONAME: libKF5GuiAddons.so.5 <-> kguiaddons>=5.6.0_1
   SONAME: libKF5ConfigGui.so.5 <-> kconfig>=5.26.0_1
   SONAME: libKF5Solid.so.5 <-> libksolid>=5.15.0_2
   SONAME: libQt5Svg.so.5 <-> qt5-svg>=5.15.2_3
   SONAME: libQt5QuickControls2.so.5 <-> qt5-quickcontrols2>=5.15.2_3
   SONAME: libQt5QuickWidgets.so.5 <-> qt5-declarative>=5.15.2_3
   SONAME: libQt5Quick.so.5 <-> qt5-declarative>=5.15.2_3
   SONAME: libQt5Concurrent.so.5 <-> qt5-concurrent>=5.15.2_3
   SONAME: libQt5Multimedia.so.5 <-> qt5-multimedia>=5.15.2_3
   SONAME: libQt5NetworkAuth.so.5 <-> qt5-networkauth>=5.15.2_3
   SONAME: libmlt.so.6 <-> mlt>=6.4.1_1
   SONAME: libKF5FileMetaData.so.3 <-> kfilemetadata5>=5.26.0_1
   SONAME: libKF5PurposeWidgets.so.5 <-> purpose>=5.48.0_1
   SONAME: libKF5Purpose.so.5 <-> purpose>=5.48.0_1
   SONAME: libKF5CoreAddons.so.5 <-> kcoreaddons>=5.26.0_1
   SONAME: libKF5ConfigCore.so.5 <-> kconfig>=5.26.0_1
   SONAME: libKF5I18n.so.5 <-> ki18n>=5.26.0_1
   SONAME: libQt5Qml.so.5 <-> qt5-declarative>=5.15.2_3
   SONAME: libQt5Network.so.5 <-> qt5-network>=5.15.2_3
   SONAME: libm.so.6 <-> glibc>=2.32_1
   SONAME: ld-linux-x86-64.so.2 <-> glibc>=2.32_1
```

![Screenshot_20210713_090044](https://user-images.githubusercontent.com/45872139/125378752-7bc69980-e3b9-11eb-870e-ea67321a8971.png)

Reply:
Yes, that's the idea, to not unnecessarily duplicate them :)

Reply:
Ok. thanks @ericonr. is there anything else that needs to be fixed?